json_schemas/brief-responses-digital-marketplace-digital-professionals.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "additionalProperties": false,
+  "additionalProperties": true,
```

Comment:
shouldn't this be false?

Reply:
this was the only way I could get existing specialists with new specialists working.  Once there are no old specialists briefs live we can set this back to `false`

Reply:
I've created MAR-1638 to track this